python/pfs/drp/stella/reduceExposure.py

```diff
@@ -199,16 +199,18 @@ def run(self, sensorRefList):
 
         fiberTraceList = [sensorRef.get("fibertrace") for sensorRef in sensorRefList]
         detectorMapList = [sensorRef.get("detectormap") for sensorRef in sensorRefList]
-        pfsConfig = sensorRefList[0].get("pfsConfig")
+        pfsConfig = None
 
         results.fiberTraceList = fiberTraceList
         results.detectorMapList = detectorMapList
-        results.pfsConfig = pfsConfig
 
         if self.config.doSubtractSky2d:
+            pfsConfig = sensorRefList[0].get("pfsConfig")
```

Comment:
Don't we need the pfsConfig whenever we are extracting spectra or solving for the wavelength solution?  It's the only way that we know which fibres are illuminated.   Yes, we can guess but now that we have pfsConfigs from LAM I think we should expect to use them.

Reply:
The `pfsConfig` tells you what the fiber is pointing at. For processing arcs, all the fibers are pointing at the same thing. The difference comes when we start doing odd/even exposures, but the pipeline will need to have a slight redesign to deal with that.
Currently, the way we know which fibers are illuminated is through the fiberTrace.

Reply:
But it also tells you which fibers are illuminated, so the findTraces procedure would not have to guess (and it is a difficult guess to make for such spare "pluggings")? I think it is important to be able to trust the fiberTrace list.

Reply:
Sure, but that involves getting the `pfsConfig` into `constructFiberTrace.py`, which is a different feature.